src/components/travel-search-filter/types.ts

```diff
@@ -0,0 +1,1643 @@
+import { z } from 'zod';
+
+export declare const TransportModeFilterOptionType: z.ZodObject<
```

Comment:
I reused this from config-specs which is used in the app. I don't have access to that repo, so this is taken from code inspection. Are there any simpler versions of this somewhere?

Reply:
Any reason to not use the package instead? 🤔  `yarn add @atb-as/config-specs`

Reply:
I guess we can. Not familiar with it. Is it public?

Reply:
It's not public. I think @mikaelbr is the owner and can add you (or the AtB Utvikling team)

Reply:
The repo isn't public, that I knew. But the package itself. Do we need to set up some auth to be able to use the package or is it public?

Reply:
`[x] Travel Search Filters`, perfect.

@mikaelbr gets to be the decision maker for using this package or not.

Reply:
This package is intended to ensure contracts between firestore and clients. So should use it here! It is public and you can use it as any other package.

Reply:
Ok! Adding the package and using it then.